tasklist/app/views/tasks/index.html.erb

```diff
@@ -1,28 +1,40 @@
 <h1>Task List</h1>
+
 <table>
-	<th colspan="2">Completed?</th><th>Task Name</th><th colspan="2">Update!</th>
+	<th colspan="2">
+		Completed?
+	</th>
+	<th>
+		Task Name
+	</th>
+	<th>
+		Owner
+	</th>
+	<th colspan="2">
+		Update!
+	</th>
+
 	<% @tasks.each do |task| %>
 	  <tr>
     	<% if task.complete_date.nil? %>
     		<td class="status" id="checkbox"><%= link_to "&#9744;".html_safe, "/tasks/complete/#{task.id}", method: :patch %></td>
+    		<td class="status">👎</td>
+    		<td class="name"><%= link_to "#{task.name}", "/tasks/#{task.id}" %></td>
     	<% else %> 
     		<td class="status" id="checkbox"><%= link_to "&#9745;".html_safe, "/tasks/incomplete/#{task.id}", method: :patch %></td>
+    		<td class="status"><%= distance_of_time_in_words(task.complete_date, Time.now) %> ago</td>
```

Comment:
Because you're using `distance_of_time_in_words(task.complete_date, Time.now)` in a couple of different view templates here, I would consider putting this into a "helper" method (inside of `app/helpers/task_helper.rb` maybe).